config/config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
   cookiebot: {
     scriptUrl: process.env.COOKIEBOT_SCRIPT_URL || '//assets.nhs.uk/scripts/cookie-consent.js',
   },
+  headerFooterApi: {
+    cacheTimeout: process.env.HEADER_FOOTER_API_CACHE_TIMEOUT || 300,
+    endpoint: process.env.HEADER_FOOTER_API_ENDPOINT || 'https://nhsuk-live.nhswebsite-integration.nhs.uk/content-api/header-footer.json',
```

Comment:
Apparently we need to use the API management URLs, the one for int is: https://nhsuk-apim-int-uks.azure-api.net/menu I am told, if there is already a API subscription key in the app that can be used.

Reply:
I imagine the API subscription key in the application will be for a specific API product. This makes it quite difficult for anybody to pick this up and work on it as an open source project without having access to the subscription key

Reply:
I don't know how it works but i was under the impression one subscription key could be used to access multiple endpoints? 

Where does open source come in to this?

Reply:
When I develop on these Node apps locally and want to view results I use my own personal "all products trial subscription" key from the API developer portal. I have never seen anything to suggest new endpoints are covered by that subscription.

In terms of open source I would expect a developer to be able to clone and run this application quickly and easily. However developers will now need to get hold of a key for it to run at all.

Reply:
Ok i see, integration doesn't require a subscription key so you should have no problem running locally using that.

Reply:
Ahhh cool thats perfect then... I will set the default to that and get the headers set up for subscription-key for other environments 👍